contrib/charts/cert-manager/templates/deployment.yaml

```diff
@@ -1,36 +1,41 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta1
```

Comment:
I think we may need another switch here to check if apps/v1beta1 exists before we try and create a resource (in order to support 1.6 deployments)

Reply:
It's already in 1.6. See https://v1-6.docs.kubernetes.io/docs/api-reference/v1.6/#deployment-v1beta1-apps